CMake/BuildFlatBuffers.cmake

```diff
@@ -325,8 +337,15 @@ function(flatbuffers_generate_headers)
     PREFIX "Flatbuffers/Schemas"
     FILES ${FLATBUFFERS_GENERATE_HEADERS_SCHEMAS})
   if (NOT ${FLATBUFFERS_GENERATE_HEADERS_BINARY_SCHEMAS_DIR} STREQUAL "")
+
+    # Resolve any relative paths for the source group call
+    get_filename_component(BINARY_SCHEMAS_DIR_ABSOLUTE_PATH
```

Comment:
Bugfix: If a relative path was passed to BINARY_SCHEMAS_DIR, the call to source_group() would cause a fatal error.  This makes the path absolute.